.circleci/config.yml

```diff
@@ -1,54 +1,7 @@
 version: 2.1
 orbs:
-  rn: react-native-community/react-native@6.0.1
+  rn: react-native-community/react-native@6.1.0
 commands:
-  yarn_install_example:
-    steps:
-      - run:
-          command: >
-            mkdir -p ~/.tmp/checksumfiles
-
-            find . -type f -name 'package.json' -not -path "*node_modules*" -exec
-            cat {} + >> ~/.tmp/checksumfiles/package.json
-
-            find . -type f -name 'yarn.lock' -not -path "*node_modules*" -exec cat
-            {} + >> ~/.tmp/checksumfiles/yarn.lock
-          name: Create cache checksum file
-
-      # Commented out because something is up with the permissions of the cache folder in the machines.
-      # There's an open issue here:  react-native-community/react-native-circleci-orb#66
-
-      # - restore_cache:
-      #     keys:
-      #       - >-
-      #         yarn-cache-{{ arch }}-{{ checksum "~/.tmp/checksumfiles/package.json"
-      #         }}-{{ checksum "~/.tmp/checksumfiles/yarn.lock" }}-{{
-      #         .Environment.CACHE_VERSION }}
-      - run:
-          command: >-
-            yarn --cwd examples/purchaseTester && yarn install --non-interactive --cache-folder /tmp/yarn
-          name: Yarn Install
-      # - save_cache:
-      #     key: >
-      #       yarn-cache-{{ arch }}-{{ checksum "~/.tmp/checksumfiles/package.json"
-      #       }}-{{ checksum "~/.tmp/checksumfiles/yarn.lock" }}-{{
-      #       .Environment.CACHE_VERSION }}
-      #     paths:
-      #       - /tmp/yarn
-
-  install_pods:
-    steps:
-      - restore_cache:
-          keys:
-            - cache-pods-{{ checksum "examples/purchaseTester/ios/Podfile.lock" }}-{{ .Environment.CACHE_VERSION }}
-            - cache-pods-
-      - run:
-          command: yarn pods
-          name: Pod install
-      - save_cache:
-          paths:
-            - examples/purchaseTester/ios/Pods
-          key: cache-pods-{{ checksum "examples/purchaseTester/ios/Podfile.lock" }}-{{ .Environment.CACHE_VERSION }}
 
 jobs:
   analyse_js:
```

Comment:
should we rename this now that it's not just js?

Reply:
I started to rename this to just `analyse` but it then made it unclear what layer it was analyzing 🤔 Even though its not 100% correct and TypeScript is a superset of JavaScript, I think it feels less worse this way 😅

Reply:
how about `analyze_ts_and_js` or `analyze_ts_js`